src/splitInteger.test.js

```diff
@@ -4,18 +4,26 @@ const splitInteger = require('./splitInteger');
 
 test(`should split a number into equal parts
   if a value is divisible by a numberOfParts`, () => {
-
+  expect(splitInteger(8, 2)).toEqual([4, 4]);
+  expect(splitInteger(10, 2)).toEqual([5, 5]);
+  expect(splitInteger(18, 3)).toEqual([6, 6, 6]);
+  expect(splitInteger(28, 4)).toEqual([7, 7, 7, 7]);
+  expect(splitInteger(0, 5)).toEqual([0, 0, 0, 0, 0]);
```

Comment:
We have the last test for "0" case. It would be better to delete this line.

Reply:
I think we can leave it here because it checks if the function properly divides zero into 5 parts.

Reply:
Ok. Let it be. At least it works)